### PR TITLE
Add pyproject.toml for Custom Node Registry

### PR DESCRIPTION
**Pull request by haohaocreates**
We are working with dr.lt.data and comfyanon to build a global registry for custom nodes (similar to PyPI). Eventually, the registry will be used as a backend for the UI-manager. All nodes go through a verification process before being published to users.

The main benefits are that authors can
- publish nodes by version and users can safely update nodes knowing ahead of time if their workflows will break or not
- automate testing against new commits in the comfy repo and existing workflows through our CI/CD dashboard

Action Required:

- [x] Go to the [registry](https://registry.comfy.org). Login and create a publisher id. Add the publisher id into the pyproject.toml file.
- [x] Write a short description.
- [x] Merge the separate Github Actions PR and run the workflow.

If you want to publish the node manually, [install the cli](https://docs.comfy.org/comfy-cli/getting-started#install-cli) and run `comfy node publish`

Check out our [docs](https://docs.comfy.org/registry/overview#introduction) if you want to know more about the registry. Otherwise, feel free to message me on discord at haohao_81202 or join our [server](https://discord.com/invite/comfyorg) if you have any questions!

### CODE REVIEW

**Comment by haohaocreates** [comment]
hey @wmatson, wanted to send a small nudge and ask if you had any questions!

**Comment by wmatson** [comment]
> hey @wmatson, wanted to send a small nudge and ask if you had any questions!

Hello @haohaocreates thanks for putting this together! I'll take a look at it over the week. I just got back from being out of country :)

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+version = "1.0.0"
+license = "LICENSE"
+dependencies = ["requests", "boto3", "rembg", "pillow-avif-plugin", "pillow-heif"]
+
+[project.urls]
+Repository = "https://github.com/wmatson/easy-comfy-nodes"
+#  Used by Comfy Registry https://comfyregistry.org
+
+[tool.comfy]
+PublisherId = ""
```
Comment (wmatson) [inline pyproject.toml]:
```suggestion
PublisherId = "wmatson"
```

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[project]
+name = "easy-comfy-nodes"
+description = "Nodes: HTTP POST, Empty Dict, Assoc Str, Assoc Dict, Assoc Img, Load Img From URL (EZ), Load Img Batch From URLs (EZ), Video Combine + upload (EZ), ..."
+version = "1.0.0"
+license = "LICENSE"
```
Comment (wmatson) [inline pyproject.toml]:
```suggestion
description = "A collection of utility nodes primarily for interacting with comfy via automated systems"
version = "1.0.0"
license = "MIT"
```

**Review by wmatson — APPROVED** [review]

**Pull request merged**